nuget/test/test-v141.vcxproj

```diff
@@ -71,6 +72,7 @@
       <SDLCheck>true</SDLCheck>
       <PreprocessorDefinitions>NDEBUG;_CONSOLE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <ConformanceMode>true</ConformanceMode>
+      <AdditionalIncludeDirectories>$(SolutionDir)\..\..\ext_libs\date;$(SolutionDir)\..\..\ext_libs\vowpal_wabbit\vowpalwabbit;$(SolutionDir)\..\..\ext_libs\vowpal_wabbit\explore;$(SolutionDir)\..\..\ext_libs\vowpal_wabbit\ext_libs\rapidjson\include;$(SolutionDir)\..\..\ext_libs\vowpal_wabbit\explore;$(SolutionDir)\..\..\ext_libs\zstd\lib;$(SolutionDir)\..\..\ext_libs\vowpal_wabbit\ext_libs\fmt\include;$(SolutionDir)\..\..\ext_libs\vowpal_wabbit\ext_libs\spdlog\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
The test project cannot reference includes otuside of the nuget. When the nuget is used none of these dirs are going to be available